### ISSUE TITLE
Is it possible to set status bar background to a colour other than black?

### ISSUE THREAD

**Issue by cliftwoc** [gpakosz/.tmux#376]
Hello.
 I'm having trouble theming the status bar. 

I can change status bar foreground colour, without problem. But I can't change the colour of the background of the status bar  Followed the instruction in the comments, but nothing seems to happen. Could someone please explain how to change status bar background colour. I want to change it from black to dark grey in all contexts. Thank you.

**Comment by gpakosz** [comment]
Hello @cliftwoc 👋

Can you please try the `gh-347` branch?

**Comment by cliftwoc** [comment]
Apologies for the late reply, I've been very ill. I've the background and foreground working. I think, I misunderstood the settings. 

I found the solution. 

I could not get the underlying background colour of the status bar to match up with colour in the left and right segments.

I needed to change the "base" status line colors. See .tmux.conf.local lines 89–92 section labeled: "# status line style". This is not the same as the section at lines 74–77, also labeled: "# status line style". Which, based on the content, I think should be called "# status line message style".

Once the status line colour is set. The  colour of the left and right segments can be set. Left, "# status left style": lines 189-189. Right, "# status right style": line 191-194. The key take away is that "# status line style" refers to the entire status line.

It works in gh-347, and current master branch.
